hack/lib/golang.sh

```diff
@@ -98,16 +98,13 @@ readonly KUBE_CLIENT_PLATFORMS=(
   linux/amd64
   linux/386
   linux/arm
-  darwin/amd64
-  darwin/386
-  windows/amd64
```

Comment:
ah, heh, that's just a hack. Uh, one of the packages flannel includes (netlink i think) had documented problems building on non-linux. Punted on it for a while.